lxd/images.go

```diff
@@ -109,7 +110,7 @@ func untarImage(imagefname string, destpath string) error {
 	return nil
 }
 
-type imagePostReq struct {
+type imageFromContainerPostReq struct {
```

Comment:
I commented about this against an older commit I think. This change is coming from a bad rebase, the name should be imagePostReq even since I added support for direct image copy.